3scale/porta
# [WIP] [UI Strings] Improve UI for creating a Backend Mapping rule

**What this PR does / why we need it**:
**Which issue(s) this PR fixes** 

[THREESCALE-7037 [UI Strings] Improve UI for creating a Backend Mapping rule](https://issues.redhat.com/browse/THREESCALE-7037)

**Verification steps** 

> Go to Backends > Mapping Rules
> Confirm the strings matches the microcopy.
> Click Create mapping rule
> Confirm the strings matches the microcopy.

Review [microcopy](https://docs.google.com/document/d/1GFl_1olM1nIkUSue9FBb7Ifp3C_mXGBCgE98divfz2Y/edit#heading=h.7u3xwescu466).

**Special notes for your reviewer**:
